pkg/build/apis/build/types.go

```diff
@@ -90,6 +90,9 @@ const (
 	// if the buildconfig does not specify a value.  This only applies to buildconfigs created
 	// via the new group api resource, not the legacy resource.
 	DefaultFailedBuildsHistoryLimit = int32(5)
+
+	// ExcerptLength is the maximum length of the LogSnippet on a build.
+	ExcerptLength = 5
```

Comment:
I don't really like this as a constant that people depend on.  Would just hide it in the build controller code